CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
+A migration will be performed when starting v1.4 for the first time to reprocess existing private transactions 
+and re-create the private state data in the v1.4 format. 
+
+If you have existing private transactions, see [migration details](docs/Private-Txns-Migration.md).
+
+### Additions and Improvements 
+
+* [TLS support](https://besu.hyperledger.org/en/latest/Concepts/TLS/) to secure client and server communication. 
+
+* [Multi-tenancy](https://besu.hyperledger.org/en/latest/Concepts/Privacy/Multi-Tenancy/) to enable multiple participants to use the same Besu and Orion node. 
+
+* [Plugin APIs](https://besu.hyperledger.org/en/latest/Concepts/Plugins/) to enable building of Java plugins to extend Hyperledger Besu. 
+
+* Support for additional [NAT methods](https://besu.hyperledger.org/en/latest/HowTo/Find-and-Connect/Specifying-NAT/).
+
+* Added [`priv_call`](https://besu.hyperledger.org/en/latest/Reference/API-Methods/#priv_call) which invokes 
+a private contract function locally and does not change the private state.
+
+* Besu has moved from an internal Bytes library to the [Apache Tuweni](https://tuweni.apache.org/) Bytes library.  
+This includes using the library in the Plugins API interfaces. [#295](https://github.com/hyperledger/besu/pull/295) and [#215](https://github.com/hyperledger/besu/pull/215)
+
+### Early Access Features 
+
+Early access features are available features that are not recommended for production networks and may 
+have unstable interfaces. 
+
+* [Reorg compatible privacy](https://besu.hyperledger.org/en/latest/Concepts/Privacy/Privacy-Overview/#reorg-compatible-privacy) 
+to enable private transactions on networks using consensus mechanisms that fork. 
+
+* [Tracing API](https://besu.hyperledger.org/en/latest/Concepts/Transactions/Trace-Types) to obtain detailed information about transaction processing. 
+
+### Bug Fixes 
+
+See RC and Beta sections below. 
+
+### Known Issues 
+
+#### Fast sync defaulting to full sync 
+
+-  When fast sync cannot find enough valid peers rapidly enough, Besu defaults to full sync. 
+
+Workarounds: 
+1. To re-attempt fast syncing rather than continue full syncing, stop Besu, delete your database, 
+and start again. 
+2. When fast syncing, explicitly disable pruning using `--pruning-enabled=false` to reduce the likelihood 
+of encountering the pruning bug. 
+
+A fix to remove the default to full sync is [in progress](https://github.com/hyperledger/besu/pull/427) 
+and is planned for inclusion in v1.4.1. 
+
+#### Error full syncing with pruning 
+
+- Error syncing with mainnet on Besu 1.3.7 node - MerkleTrieException [\#BESU-160](https://jira.hyperledger.org/browse/BESU-160)
+The associated error is `Unable to load trie node value for hash` and is caused by the combination of 
+full sync and pruning. 
+
+Workarounds: 
+1. Explicitly disable pruning using `--pruning-enabled=false` when using fast sync. 
```

Comment:
This should either say full sync or just disable in general. We don't actually know how likely it is to occur for fast sync but the pruning times are spread out enough that it's definitely a much smaller issue than full sync.